packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -140,7 +138,7 @@ class SocketReference extends TypedEventEmitter<ISocketEvents> {
         }
     }
 
-    private closeSocket() {
+    public closeSocket() {
         if (!this._socket) { return; }
```

Comment:
(Continuation of comment in disposeSocket()):
should we ever get here with this.references === 0? And this be true when promise on line 158 runs?

Reply:
We should be getting here when this.references === 0. What is the problem in that?